iredis/entry.py

```diff
@@ -261,6 +439,7 @@ def main():
     client = Client(
         ctx.params["h"], ctx.params["p"], ctx.params["n"], ctx.params["password"]
     )
+
```

Comment:
I think we should resolve the `ctx` for client args here, and pass the arguments explictly.